Projects/Server/Gumps/Gump.cs

```diff
@@ -15,7 +15,19 @@ public partial class Gump
     public static readonly byte[] NoDispose = StringToBuffer("{ nodispose }");
     public static readonly byte[] NoResize = StringToBuffer("{ noresize }");
 
-    internal int m_TextEntries, m_Switches;
+    private int m_Switches, m_TextEntries;
```

Comment:
Let's rename them to use the new convention of _switches, etc.

Reply:
Done!